CMakeLists.txt

```
@@ -1,73 +1,76 @@
-cmake_minimum_required(VERSION 3.22.1)
+cmake_minimum_required(VERSION 3.23)
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/CMake")
-include(ystdlib-cpp-helpers)
+project(ystdlib VERSION "0.0.1" LANGUAGES CXX)
```

```suggestion
project(ystdlib VERSION "0.1.0" LANGUAGES CXX)
```

For new projects, we start our versioning at 0.1.0.